### DISCUSSION TITLE
Is there a way to change how long it takes until a node is considered disconnected? Also, can we control wifi power saving mode?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [ESPresense/ESPresense#1039] [Q&A]
I have a large number of nodes, and even the ones that are POE are very frequently "disconnecting" for a second or two. Is there a way to increase the amount of time before a node is considered disconnected? I'd also like to know if there is a way to control wifi power saving mode. I'd like to be able to set it to "none" from the default ESP32 setting of "light" if that's possible. I think this would also help my wifi nodes constantly disconnecting and reconnecting.

**Comment by fallingaway24** [comment]
do you have issues with nodes not updating very quick? I've been tinkering with espresense for about a year and just curious how others are faring. If you've found any tips or tricks to help with a multi-node/multi-hub setup.

**Comment by Terastar-Paperclip** [comment]
Rolled this into our canonical "away detection" writeup as the WiFi/MQTT-flap case — a brief disconnect fires the node's LWT `offline`, which downstream can read as every device on that node going away. Worth checking PoE/AP-side EEE negotiation and your broker's keepalive. Full pin: #2322. The WiFi power-save UI toggle you're asking about is still an open ask (noted in the pin) — no firmware setting for that today; flagging it with the core team since it comes up periodically.
